model/ftn/ww3_uprstr.ftn

```diff
@@ -827,46 +1167,176 @@
 !     !/T
 !
 ! 10. Source code :
-         
+!         
+!/
          REAL, DIMENSION(:,:), INTENT(OUT) :: UPDPRCNT
          CHARACTER(*), INTENT(IN) :: FLNMCOR
+         LOGICAL, INTENT(IN) :: SMCGRD
```

Comment:
SMCGRD available here from prorgam level declaration?

Reply:
Following discussion - not changing this for the time being, but scope to be updated if adding other grid types to the ww3_uprstr.ftn methods